ex2_hal/iris/equipment_handler/include/iris_spi.h

```diff
@@ -30,6 +30,7 @@ typedef enum {
 } IrisLowLevelReturn;
 
 #define DUMMY_BYTE 0xFF
+#define IRIS_WAIT_FOR_ACK vTaskDelay(20)
```

Comment:
Anther thing is the numbers here are unrelated to any sort of time. It is a delay in ticks. So you should use pdMS_TO_TICKS(n)

Reply:
Correct, I have updated the file